env/.env.dev.example

```
@@ -0,0 +1,31 @@
+# dev placeholder source envelope
+GOVERNANCE_POLICY_SOURCE=./config/governance-policy.dev.json
+# dev placeholder HMAC key
+GOVERNANCE_POLICY_SOURCE_HMAC_KEY=dev-hmac-placeholder
+
+# Observability
+OTEL_SERVICE_NAME=file-engine-api
+OTEL_EXPORTER_OTLP_ENDPOINT=http://otel-collector:4317
+
+# Audit sink (dev-safe local file)
+AUDIT_IMMUTABLE_SINK_TYPE=file
+AUDIT_IMMUTABLE_SINK_PATH=/tmp/audit.ndjson
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUDIT_IMMUTABLE_SINK_PATH key should go before the AUDIT_IMMUTABLE_SINK_TYPE key

(UnorderedKey)